### DISCUSSION TITLE
Disable ragdolling

### DISCUSSION THREAD

**Discussion by MostSinister** [Sixze/ALS-Refactored#436] [General]
Any one knows of a way i can prevent ragdoll from starting? My character starts ragdolling when it hits the water surface too fast. I tried changing collision object type and trace setting on the water but it didnt help.

Thx yall

**Comment by MrWilsonWilson** [comment]
Check in the data assets there are settings out of the box prepared for that !
![image](https://github.com/Sixze/ALS-Refactored/assets/122884397/00eb6b60-4ce2-4f84-9bfb-8eda74d11fcc)


CS_Als_Default

**Comment by MostSinister** [comment]
Yeah I have tried Before, it allows me to turn off ragdolling altogether but i did not find a way to change this dynamically from a blueprint during gameplay. Specifically, enabling/disabling the "start ragdolling on land" or something to that effect. i dont think it is exposed or that there is a build in function for that.

it would be kind of a sketchy workaround anyway so i'll fix the collision and trace channels in my project first. But i am still curious how someone would go about doing that. 

Thank for your help!
